### ISSUE TITLE
Add cors support to s3_bucket module

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29238]
From @bermudezjd on 2015-10-26T18:46:09Z
##### ISSUE TYPE

Feature Request
##### COMPONENT NAME

s3_bucket
##### ANSIBLE VERSION

N/A
##### SUMMARY

Add cors support to s3_bucket module
@wimnat : mind if I take a crack at adding cors xml support in my fork for an eventual PR?

Copied from original issue: ansible/ansible-modules-extras#1163

**Comment by ansibot** [comment]
From @bermudezjd on 2015-10-26T18:46:09Z

Hi @bermudezjd.  I did do some work on cors support.  I'll see if I can pull it out.

I wouldn't add it to s3_bucket.  I think my plan was a new module s3_cors_rule which added and removed individual rules as necessary.  I think this is cleaner that trying to add all your cors rules in one module.

PS you might want to move this conversation to ansible devel google group.

**Comment by ansibot** [comment]
From @bermudezjd on 2015-10-26T18:46:09Z

Linking PR https://github.com/ansible/ansible-modules-extras/pull/1177

**Comment by ansibot** [comment]
From @bermudezjd on 2015-10-26T18:46:09Z

@bermudezjd ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @bermudezjd on 2015-10-26T18:46:09Z

@bermudezjd ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @bermudezjd on 2015-10-26T18:46:09Z

Not sure if it's worth reconfiguring this PR.  I've already done so 3 or 4 times and it never seems to get approved.  Any reason why I should put this out again?

**Comment by ansibot** [comment]
cc @willthames
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
